frontend/public/css/style.css

```diff
@@ -6,6 +6,12 @@
     top:3px;
 }
 
+.col-md-12 {
+    margin-top: 10px;
+    margin-bottom: 30px;
+    padding-bottom: 20px;
```

Comment:
Can you make this footer specific as we might have problems on other pages as this is a site wide style.css

Reply:
@abishekvashok i am not getting your point.will you explain more

Reply:
This change is applicable to all elements of the col-md-12 class. That's unwanted. Can you narrow the changes to footer only?